packages/client/src/components/connect-wallet-modal.tsx

```diff
@@ -62,18 +57,18 @@ function ConnectWalletModal({
                     <div className='connect-wallet-modal-options-container'>
                         <button
                             className='connect-wallet-modal-option'
-                            disabled={!availableProviders.metamask}
+                            // disabled={!availableProviders.metamask}
```

Comment:
What happens if the user doesn't have the Metamask extension installed? I.e. they have no options here?